cumulusci/cli/cci.py

```diff
@@ -725,11 +730,24 @@ def project_info(runtime):
     name="dependencies",
     help="Displays the current dependencies for the project.  If the dependencies section has references to other github repositories, the repositories are inspected and a static list of dependencies is created",
 )
+@click.option(
+    "--resolution-strategy",
+    help="The resolution strategy to use. Defaults to production.",
+    default="production",
+)
 @pass_runtime(require_keychain=True)
-def project_dependencies(runtime):
-    dependencies = runtime.project_config.get_static_dependencies()
-    for line in runtime.project_config.pretty_dependencies(dependencies):
-        click.echo(line)
+def project_dependencies(runtime, resolution_strategy):
+    dependencies = get_static_dependencies(
+        parse_dependencies(runtime.project_config.project__dependencies),
+        get_resolver_stack(
+            runtime.project_config,
+            resolution_strategy,
+        ),
+        runtime.project_config,
+    )
```

Comment:
Calling get_static_dependencies looks more complicated than it needs to be -- the dependencies input and resolver stack can be derived from the project_config in cases where they aren't being overridden to come from somewhere else, no?